lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -1493,6 +1502,7 @@ terminator('(')  -> ')';
 terminator('[')  -> ']';
 terminator('{')  -> '}';
 terminator('"""') -> '"""';
+terminator('"')  -> '"';
```

Comment:
I have been thinking for a while that we should include the mismatched_delimiter and expected_delimiter directly in the exception struct. The fact we are adding new entries here make me thing we should go ahead with adding these fields. WDYT?

Reply:
I guess that's fine, I see no problem in doing it this way as well, but perhaps by also adding it to the exception makes it a bit more complete

In the end I think that the least work we have to do in the exception code and can do it before is better, because the overall diagnostics get closer to the diagnostics we print to the terminal as well (although in some cases it will probably better to just keep them different)